SPECS/openssh/openssh.spec

```diff
@@ -16,6 +16,7 @@ Patch0:         blfs_systemd_fixes.patch
 # Nopatches section
```

Comment:
Looks like this was intended for nopatch only, the real patches should be above (`Patch1: CVE-2023-28531-upstream-include-destination-constraints-for-smartca.patch`)

Reply:
does it matter?